debian/changelog

```diff
@@ -1,3 +1,9 @@
+cloud-init (22.2-64-g1fcd55d6-0ubuntu1~22.10.2) UNRELEASED; urgency=medium
+
+  * Fix lintian warnings
```

Comment:
When we touch debian/* packaging files we generally want to refer to the specific files changed and the reason behind it (as SRU tend to look more closely at the packaging related changes) and it is more intuitive for review to  see those packaging changes called out specifically

So we need to see something like the specific warning handled and the reason for each file
* d/source/lintian-overrides:
  + silence binary-nmu-debian-revision-in-source as deb ver includes series
* d/control: lintian fixes: drop -XS-Python-Version  and XB-Python-Version because XXX
* d/cloud-init.lintian-overrides:
  + format of systemd-service-file-refers-to-unusual-wantedby-target changed?
* d/source/cloud-init.postinst: avoid hard-coded paths for grub-install

Reply:
Done.